Mage/src/main/java/mage/abilities/effects/common/FightTargetsEffect.java

```diff
@@ -16,8 +16,15 @@
  */
 public class FightTargetsEffect extends OneShotEffect {
 
+    protected boolean showEffectHint;
+
     public FightTargetsEffect() {
-        super(Outcome.Damage);
```

Comment:
Effect's outcome (bad or good) used by AI to activate ability or make a targeting/choices So it must be changed with carefully. Just for info.

Reply:
Should I change it back to Benefit?

Reply:
Nope. If AI will be bugged with that effect then it can be changed back. I think current code can improve some effect's usage by AI (in theory, I don't test it).